compiler/tflchef/tests/short_int4/test.recipe

```diff
@@ -0,0 +1,44 @@
+operand {
+  name: "ifm"
+  type: INT4
+  shape { dim: 1 dim: 5 dim: 5 dim: 1 }
+}
+operand {
+  name: "ker"
+  type: INT4
+  shape { dim: 1 dim: 3 dim: 3 dim: 1 }
+  filler {
+    tag: "explicit"
+    arg: "-1" arg: "0" arg: "1"
+    arg: "1" arg: "2" arg: "3"
+    arg: "-1" arg: "-2" arg: "-3"
+  }
+}
+operand {
+  name: "bias"
+  type: INT4
```

Comment:
tflite supports convolution with ifm int8, ker int4, bias int32.

Reply:
ah, this recipe is just to validate int4 not related with tflite.

Reply:
tflite recipes in `/res` will have actual data types... I hope :)